src/vl-map-action.js

```diff
@@ -23,18 +32,27 @@ export class VlMapAction extends vlElement(HTMLElement) {
   }
 
   /**
-   * Geeft de event naam die gebruikt wordt wanneer een nieuwe actie toegevoegd wordt aan de kaart
+   * Geeft de stijl van de kaartinteractie.
+   *
+   * @return {Object}
+   */
+  get style() {
+    return this._style;
+  }
+
+  /**
+   * Zet de stijl van de kaartinteractie.
    *
-   * @return {string}
+   * @param {Object} style
```

Comment:
Deze documentatie (en ook hieronder) brengt naar mijn inziens eigenlijk niet zo veel bij. Op zich vind ik ook wel vreemd dat een MapAction altijd een stijl heeft (is dit niet eerder bij specifieke gevallen zoals SelectAction?).

Reply:
Ik heb de stijl verplaatst naar de vl-map-select-action en de documentatie aangevuld.